modules/openapi/tests/resources/issue-23.json

```diff
@@ -15,7 +15,7 @@
     }
   ],
   "paths": {
-    "/Testers/{id}": {
+    "/Testers/{id_}": {
```

Comment:
bumping alloy/smithy has had the side effect of fixing false-positive validation errors we were actually counting on to test. So I'm injecting new errors to keep the tests working.